docs/index.md

```diff
@@ -5428,6 +5430,63 @@ return api.sign(txJSON, secret); // or: api.sign(txJSON, keypair);
 }
 ```
 
+### Example (multisigning)
+
+```javascript
+const RippleAPI = require('ripple-lib').RippleAPI;
+const api = new RippleAPI({
+  server: 'wss://s.altnet.rippletest.net:51233' 
+});
+
+// jon's address has a multi-signing setup with a quorum of 2
+const jon = {
+  account: 'ranYTqDRUKKAqnXAAVDwWBUvvc5op4pXoP',
+  secret: 'shfC5QySdfhCUaEfEF9gxPhAhy58u'
+};
+
+const aya = {
+  account: 'rnrPdBjs98fFFfmRpL6hM7exT788SWQPFN',
+  secret: 'snaMuMrXeVc2Vd4NYvHofeGNjgYoe'
+};
+
+const bran = {
+  account: 'rJ93RLnT1t5A8fCr7HTScw7WtfKJMRXodH',
+  secret: 'shQtQ8Um5MS218yvEU3Ehy1eZQKqH'
+};
+
+// a transaction which requires multi signing
+const multiSignPaymentTransaction = {
+    TransactionType: 'Payment',
+    Account: 'ranYTqDRUKKAqnXAAVDwWBUvvc5op4pXoP',
+    Sequence: 4,
+    Destination: 'rJ93RLnT1t5A8fCr7HTScw7WtfKJMRXodH',
+    Fee: '120',
+    Amount: '1888000000'
+};
+
+api.connect().then(() => {
+  api.prepareTransaction(multiSignPaymentTransaction).then(prepared => {
+      console.log(prepared);
+      
+      // Aya and Bran sign it too but with 'signAs' set to their own account
+      let ayaSign = api.sign(tx, aya.secret, {'signAs': aya.account}).signedTransaction;
+      let branSign = api.sign(tx, bran.secret, {'signAs': bran.account})).signedTransaction;
+      
+      // signatures are combined and submitted
+      let combinedTx = api.combine([evaSig, giliSig, sincanSig]);
```

Comment:
I don't see these defined anywhere: `evaSig, giliSig, sincanSig`

Reply:
@intelliot Sorry that was a typo and fixed.

Ok, I added the SignerListSet transaction and replaced all JSON.stringify with prepareTransaction.
But I got this error:
`[MissingLedgerHistoryError(Server is missing ledger history in the specified range)]
`

I read about the similar issues on the community forum and here and even tried to get the min and max ledger version from the server -via getServerInfo()- and manually convert them from String to Number. But I still get that error.
 
Here is the sample code.
```
const RippleAPI = require('ripple-lib').RippleAPI;

// jon's address has a multi-signing setup with a quorum of 2
const jon = {
    account: 'rJKpme4m2zBQceBuU89d7vLMzgoUw2Ptj',
    secret: 'sh4Va7b1wQof8knHFV2sxwX12fSgK'
};
const aya = {
    account: 'rnrPdBjs98fFFfmRpL6hM7exT788SWQPFN',
    secret: 'snaMuMrXeVc2Vd4NYvHofeGNjgYoe'
};
const bran = {
    account: 'rJ93RLnT1t5A8fCr7HTScw7WtfKJMRXodH',
    secret: 'shQtQ8Um5MS218yvEU3Ehy1eZQKqH'
};

// Setup the signers list with a quorum of 2
const multiSignSetupTransaction = {
    "Flags": 0,
    "TransactionType": "SignerListSet",
    "Account": "rJKpme4m2zBQceBuU89d7vLMzgoUw2Ptj",
    "Fee": "120",
    "Sequence": 1,
    "SignerQuorum": 2,
    "SignerEntries": [
        {
            "SignerEntry": {
                "Account": "rnrPdBjs98fFFfmRpL6hM7exT788SWQPFN",
                "SignerWeight": 2
            }
        },
        {
            "SignerEntry": {
                "Account": "rJ93RLnT1t5A8fCr7HTScw7WtfKJMRXodH",
                "SignerWeight": 1
            }
        },
    ]
};

// a transaction which requires multi signing
const multiSignPaymentTransaction = {
    TransactionType: 'Payment',
    Account: 'rUGyyqhHZtZeti1gY9gd8g4Pcss6PQNqUH',
    Destination: 'rJ93RLnT1t5A8fCr7HTScw7WtfKJMRXodH',
    Amount: '88000000'
};

const api = new RippleAPI({
    server: 'wss://s.altnet.rippletest.net:51233'
});

api.connect().then(() => {
    // adding the multi signing feature to jon account
    api.prepareTransaction(multiSignSetupTransaction).then((prepared) => {
        console.log(prepared);
        jonSign = api.sign(prepared.txJSON, jon.secret).signedTransaction;
        api.submit(jonSign).then( response => {
            console.log(response.resultCode, response.resultMessage);

            // multi sign a transaction
            api.prepareTransaction(multiSignPaymentTransaction).then(prepared => {
                console.log(prepared);

                // Aya and Bran sign it too but with 'signAs' set to their own account
                let ayaSign = api.sign(prepared.txJSON, aya.secret, {'signAs': aya.account}).signedTransaction;
                let branSign = api.sign(prepared.txJSON, bran.secret, {'signAs': bran.account}).signedTransaction;

                // signatures are combined and submitted
                let combinedTx = api.combine([ayaSign, branSign]);
                api.submit(combinedTx.signedTransaction).then(response => {
                    console.log(response.tx_json.hash);
                    api.getTransaction(response.tx_json.hash).then (response => {
                        console.log(response);
                        return api.disconnect();
                    }).catch(console.error);
                }).catch(console.error);
            }).catch(console.error);
        }).catch(console.error)
    }).catch(console.error);
}).catch(console.error);
```

I wonder if you can give me hint here so I can learn something new. Thanks man.

Reply:
I tried that on multiple test accounts and every time the settings change transaction works flawlessly... but when I submit the multi-signed payment transaction it doesn't go through (It generates a hash though)

Check [this](https://test.bithomp.com/explorer/rJKpme4m2zBQceBuU89d7vLMzgoUw2Ptj) address on test net for example.

Reply:
You know when I check the account who receives the payment, I can see the transaction is already completed and the receiver balance is increased. But that transaction is not showing in the sender account.  o_0 
I am not sure what is happening here...